loops_lab.js

```diff
@@ -0,0 +1,292 @@
+// //## Exercises
```

Comment:
Good demonstration of for and while loops. No major errors or feedback. Keep practicing and also keep up the good work!